src/test/rustdoc/issue-88600.rs

```diff
@@ -0,0 +1,18 @@
+// @has issue_88600/enum.FooEnum.html
+// @has - '//*[@id="variant.HiddenTupleItem"]//code' 'HiddenTupleItem(_)'
+// @!has - '//*[@id="variant.HiddenTupleItem.field.0"]' '0: ()'
+// @has - '//*[@id="variant.MultipleHidden"]//code' 'MultipleHidden(_, _)'
+// @!has - '//*[@id="variant.MultipleHidden.field.0"]' '0: ()'
+// @!has - '//*[@id="variant.MultipleHidden.field.1"]' '1: ()'
+// @has - '//*[@id="variant.MixedHiddenFirst"]//code' 'MixedHiddenFirst(_, ())'
+// @!has - '//*[@id="variant.MixedHiddenFirst.field.0"]' '0: ()'
+// @has - '//*[@id="variant.MixedHiddenFirst.field.1"]' '1: ()'
+// @has - '//*[@id="variant.MixedHiddenLast"]//code' 'MixedHiddenLast((), _)'
+// @has - '//*[@id="variant.MixedHiddenLast.field.0"]' '0: ()'
+// @!has - '//*[@id="variant.MixedHiddenLast.field.1"]' '1: ()'
+pub enum FooEnum {
+    HiddenTupleItem(#[doc(hidden)] ()),
```

Comment:
Sorry for the nitpicking here but: could you use different types (between hidden and non-hidden fields) so that it's much easier to see what's going on please?